### ISSUE TITLE
resizing behaviour on horizontal splits for 2 or more files

### ISSUE THREAD

**Issue by kleungkw** [fholgado/minibufexpl.vim#93]
I've just recently updated the plugin and noticed when I have horizontal splits of 2 or more files with MiniBufExplorer opened the splits get resized evenly when I switch focus in one of these windows.  This gets frustrating when I've manually resized the windows and they keep getting resized evenly.

This didn't happen before :TMiniBufExplorer was deprecated, and it doesn't seem to affect vertical splits.

Is this window resizing an intended behaviour?

**Comment by weynhamz** [comment]
Yes, this was inctroduced some time ago in order to fix #80, it indeed behave the way you decribed in your situation, it is one drawback though, there should be a way to fully resolve #80, I will be look into it.

**Comment by ghost** [comment]
It is a very bad behavior :~/ that I only found recently because I use more often vertical splits.
And it did take me some time to find that MBE was the culprit !
I would be able to change a buffer in a window without resizing anything (using an MBE option if it is necessary to preserve others functionalities).
Hope it is possible, I love MBE and thanks for your work.

**Comment by anishathalye** [comment]
This still seems to be an issue. Is there a simple way to fix it?
